src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -22,41 +23,66 @@ public JsonSchemaTransformer(JsonSchema schema)
         public (List<Error> errors, JToken token) TransformContent(Document file, Context context, JToken token, Action<Document> buildChild)
         {
             var errors = new List<Error>();
-            Traverse(_schema, token, (schema, value) => value.Replace(TransformScalar(schema, file, context, value, errors, buildChild)));
+            Traverse(_schema, token, (_, schema, name, value) => value.Replace(TransformScalar(schema, file, context, value, errors, buildChild)));
             return (errors, token);
         }
 
-        public (List<Error> errors, Dictionary<string, Lazy<JValue>>) TransformXref(Document file, Context context, JToken token)
+        public (List<Error> errors, Dictionary<string, (bool, Dictionary<string, Lazy<JValue>>)> properties) TransformXref(Document file, Context context, JToken token)
```

Comment:
```suggestion
        public (List<Error> errors, Dictionary<string, (bool isRoot, Dictionary<string, Lazy<JValue> properties>)> xrefProperties) TransformXref(Document file, Context context, JToken token)
```

Reply:
shall we follow the naming convention of the return type? it helps me to understand what these mean, especially for this kind of complex types